lerna.json

```diff
@@ -12,6 +12,7 @@
       "createRelease": "github",
       "gitDryRun": false,
       "noPrivate": true,
+      "packageLockfileOnly": true,
```

Comment:
I'd assume this flag is enabled by default when the user uses a workspace feature from the package manager.

Reply:
hmm but that is partially related to what I said in my last comment, I'm not sure if I'd make this new flag enabled by default. Also there's currently no logic in place to verify what type of workspace is in place, that's basically why there's an `npmClient` property in place and I'm not sure that I want to spend more time to deal with extra logic just to avoid the user to simply enable the flag himself. That topic actually came up in the other lock file opened PR, the solution by the other user was to use something like [preferred-pm](https://www.npmjs.com/package/preferred-pm)) to find out which is used but that package has a couple of dependencies and I'm not crazy about adding more and more dependencies... so is it worth the extra time